MarathonRecomp/patches/player_patches.cpp

```diff
@@ -465,3 +465,7 @@ void SonicGaugeRestorationGaugeFlagFix(PPCRegister& r_gauge, PPCRegister& r_cont
         }
     }
 }
+
+bool InfiniteLives() {
```

Comment:
Use Allman style indentation.

Reply:
There's some other indentation mistakes in player_patches, I'll resolve them alongside this